### ISSUE TITLE
LuaTableKeyCall sends nil args

### ISSUE THREAD

**Issue by PepsiBoyDev** [rdeioris/LuaMachine#15]
Messing around with this and I noticed that no matter how I set up LuaTableKeyCall it triggers the function correctly, but any args passed through it are nil

![image](https://user-images.githubusercontent.com/34015862/93665194-be7a5680-fab3-11ea-8613-619682a8a49a.png)

```lua
function Widget:Update(testString)
	print(testString)
end
```

I'm not entirely sure if I have just configured it wrong somewhere, but since the actual function is firing I really have no idea, ive tried it in different blueprints, global calls pass args correctly so I'm stumped.

**Comment by PepsiBoyDev** [comment]
Oop, right after submitting this I tried the same thing with the LuaTableKeyCallwithSelf and it works correctly, hower I noticed a typo.

![image](https://user-images.githubusercontent.com/34015862/93665378-1e253180-fab5-11ea-932b-dffc08353222.png)

The typo is extremely minor but its worth noting.

I'm most likely just using TableKeyCall in the wrong context (most likely after I actually read the description text), been a long time since I touched either Lua or UE4 or programming in general.
